test/inductor/test_flex_attention.py

```diff
@@ -199,6 +202,7 @@ def _check_equal(
     ):
         compiled_error = (golden_out - compiled_out).abs().mean()
         ref_error = (golden_out - ref_out).abs().mean()
+        # TODO: Make this check stricter after updating eager SDPA masked_softmax semantics
```

Comment:
After eager SDPA switching to ```_masked_softmax```(#131071), we should enhance it to check no nan for both ```compiled_error``` and ```ref_error```.